pkg/apis/batch/validation/validation.go

```diff
@@ -675,3 +820,10 @@ type JobValidationOptions struct {
 	// Require Job to have the label on batch.kubernetes.io/job-name and batch.kubernetes.io/controller-uid
 	RequirePrefixedLabels bool
 }
+
+type JobStatusValidationOptions struct {
```

Comment:
After https://github.com/kubernetes/kubernetes/pull/123273 is merged, I will rebase this PR and re-organize this function a little bit.

Reply:
This was resolved.